src/pages/payment-methods/_payment-data.tsx

```diff
@@ -66,81 +66,81 @@ const StyledRefLink = styled(LocalizedLink)`
 // `
 const payment_data = [
     {
-        name: <Localize translate_text="Online banking" />,
+        name: <Localize translate_text="_t_Online banking_t_" />,
```

Comment:
We have to refactor whole file and put `<Localize />` component in place when we shows this text
<img width="910" alt="Screenshot 2023-02-23 at 09 41 21" src="https://user-images.githubusercontent.com/103182473/220836626-a5c86d4e-735e-40b5-9f33-98e351bb81fa.png">